pvacseq/lib/call_iedb.py

```diff
@@ -46,12 +52,12 @@ def main(args_input = sys.argv[1:]):
     url = prediction_class_object.url
 
     response = requests.post(url, data=data)
-    if response.status_code == 500:
-        #Retry once
+    retries = 0
+    while response.status_code == 500 and retries < args.iedb_retries:
         response = requests.post(url, data=data)
-        if response.status_code == 500:
-            #Retry a second time
-            response = requests.post(url, data=data)
+        print("Retries %s" % retries)
```

Comment:
This is actually a leftover line of code from some debugging I did which I didn't actually want to commit. However, you suggestion seems like a good idea so I will change the status message to that.